ci/infra/azure/main.tf

```diff
@@ -13,3 +13,21 @@ data "azurerm_platform_image" "sles_chost_byos" {
   offer     = "sles-15-sp1-chost-byos"
   sku       = "gen2"
 }
+
+resource "azurerm_image" "sles_chost_byos" {
+  # name                      = "SLES15-SP2-CHOST-BYOS.x86_64-0.9.12-Azure-Build1.4.vhd"
+  name                      = "sles-15-sp2-chost-byos"
+  location                  = "West Europe"
+  resource_group_name       = "openqa-upload"
+
+    os_disk {
+    os_type  = "Linux"
+    os_state = "Generalized"
+    # blob_uri = "{blob_uri}"
```

Comment:
Please remove if not used.

Reply:
I notice I add the testing code SP2. I have removed it.